app/controllers/concerns/file_upload_limitable.rb

```diff
@@ -0,0 +1,16 @@
+module FileUploadLimitable
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :single_file_upload_limit, only: [:create]
+  end
+
+  private
+
+  def single_file_upload_limit
+    return unless params[:files].is_a?(Array)
```

Comment:
do you think using `one?` here will be easier to read?
https://apidock.com/ruby/Enumerable/one%3F

we could return if `params[:files]` only contains one element

Reply:
So it interesting how files are handled in rails and I'd have to look into it deeper but when only one file is attached `params[:files].class` returns `ActionDispatch::Http::UploadedFile`. If more than one file is attached you get an array of uploaded files. So for a single file upload `params[:files].one?` returns undefined method.

Reply:
Ahh yes that makes sense then. I thought it couldn’t be that simple based on how you had it in hindsight.